src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/GaussianPlusConstant.hpp

```diff
@@ -0,0 +1,119 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines GeneralizedHarmonic::ConstraintDamping::GaussianPlusConstant.
```

Comment:
We don't require (and prefer to not have) a `file` comment for files where the only thing implemented matches the name of the file